courses/1-ramda-basics/2-composition.js

```diff
@@ -193,8 +195,14 @@ describe("Course 1.2 - Composition", () => {
         },
       ];
 
+      const goodOmens = R.filter(R.propEq('title', 'Good Omens'));
```

Comment:
This is great! I was hoping you would try to reach for the Ramda functions to solve this one.